x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/step_define_rule/index.tsx

```diff
@@ -446,37 +462,65 @@ const StepDefineRuleComponent: FC<StepDefineRuleProps> = ({
     ]
   );
 
+  const [{ queryBar }] = useFormData<DefineStepRule>({ form, watch: ['queryBar'] });
+  const areSuppressionFieldsDisabledBySequence =
+    isEqlRule(ruleType) &&
+    isEqlSequenceQuery(queryBar?.query?.query as string) &&
+    groupByFields.length === 0;
+
   /**
    * Component that allows selection of suppression intervals disabled:
    *  - if suppression license is not valid(i.e. less than platinum)
    *  - or for not threshold rule - when groupBy fields not selected
+   * - Eql sequence is used
    */
   const isGroupByChildrenDisabled =
-    !isAlertSuppressionLicenseValid || isThresholdRule ? false : !groupByFields?.length;
+    areSuppressionFieldsDisabledBySequence || !isAlertSuppressionLicenseValid || isThresholdRule
+      ? false
+      : !groupByFields?.length;
 
   /**
    * Per rule execution radio option is disabled
    *  - if suppression license is not valid(i.e. less than platinum)
    *  - always disabled for threshold rule
+   *  - Eql sequence is used and suppression fields are in the default state
    */
-  const isPerRuleExecutionDisabled = !isAlertSuppressionLicenseValid || isThresholdRule;
+  const isPerRuleExecutionDisabled =
+    areSuppressionFieldsDisabledBySequence || !isAlertSuppressionLicenseValid || isThresholdRule;
 
   /**
    * Per time period execution radio option is disabled
    *  - if suppression license is not valid(i.e. less than platinum)
    *  - disabled for threshold rule when enabled suppression is not checked
+   * - Eql sequence is used and suppression fields are in the default state
    */
   const isPerTimePeriodDisabled =
-    !isAlertSuppressionLicenseValid || (isThresholdRule && !enableThresholdSuppression);
+    areSuppressionFieldsDisabledBySequence ||
+    !isAlertSuppressionLicenseValid ||
+    (isThresholdRule && !enableThresholdSuppression);
 
   /**
    * Suppression duration is disabled when
    *  - if suppression license is not valid(i.e. less than platinum)
    *  - when suppression by rule execution is selected in radio button
-   *  - whe threshold suppression is not enabled and no group by fields selected
+   *  - when threshold suppression is not enabled and no group by fields selected
+   * -  Eql sequence is used and suppression fields are in the default state
```

Comment:
what does default state mean in this context?

Reply:
There was previously logic in the rule form to keep the suppression fields enabled (but invalid) until they were changed back to their default values. I removed this as part of the form refactor, but this might also be the cause of the UI oddities you observed, so I'll investigate.

Reply:
Update: see https://github.com/elastic/kibana/pull/176422#discussion_r1563610751